### ISSUE TITLE
[Bug]: lottie file load error: Expected class kotlinx.serialization.json.JsonArray (Kotlin reflection is not available) as the serialized body of kotlin.collections.ArrayList, but had class kotlinx.serialization.json.JsonLiteral (Kotlin reflection is not available)

### ISSUE THREAD

**Issue by Kale1314** [alexzhirkevich/compottie#14]
### What happened?

The UI designer gave me a lottey file, but I am unable to load it

### Lottile File

[Voice circle.json](https://github.com/user-attachments/files/16437959/Voice.circle.json)

### Screenshot

![image](https://github.com/user-attachments/assets/65a9ad25-716b-406d-8dfc-e754c4011f5c)

### What Compottie version do you experience the bug on?

2.0.0-beta02

### What platforms are you seeing the problem on?

Android

### Does Compottie log anythyng to the stdout / your logger set with `Compottie.logger`?

```shell
2024-07-31 15:34:34.176 15729-15729 Verificati...okeSuspend com.demo.cm                          E  load lottie failed
2024-07-31 15:34:34.176 15729-15729 Verificati...okeSuspend com.demo.cm                          E  io.github.alexzhirkevich.compottie.CompottieException: Composition failed to load
2024-07-31 15:34:34.176 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.LottieCompositionKt$rememberLottieComposition$1.invokeSuspend(LottieComposition.kt:81)
2024-07-31 15:34:34.176 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
2024-07-31 15:34:34.176 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:102)
2024-07-31 15:34:34.176 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at androidx.compose.ui.platform.AndroidUiDispatcher.performTrampolineDispatch(AndroidUiDispatcher.android.kt:81)
2024-07-31 15:34:34.176 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at androidx.compose.ui.platform.AndroidUiDispatcher.access$performTrampolineDispatch(AndroidUiDispatcher.android.kt:41)
2024-07-31 15:34:34.176 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at androidx.compose.ui.platform.AndroidUiDispatcher$dispatchCallback$1.run(AndroidUiDispatcher.android.kt:57)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at android.os.Handler.handleCallback(Handler.java:958)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at android.os.Handler.dispatchMessage(Handler.java:99)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at android.os.Looper.loopOnce(Looper.java:224)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at android.os.Looper.loop(Looper.java:318)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at android.app.ActivityThread.main(ActivityThread.java:8767)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at java.lang.reflect.Method.invoke(Native Method)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:561)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  Caused by: kotlinx.serialization.json.internal.JsonDecodingException: Expected class kotlinx.serialization.json.JsonArray (Kotlin reflection is not available) as the serialized body of kotlin.collections.ArrayList, but had class kotlinx.serialization.json.JsonLiteral (Kotlin reflection is not available)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.JsonExceptionsKt.JsonDecodingException(JsonExceptions.kt:24)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.beginStructure(TreeJsonDecoder.kt:329)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.JsonTreeDecoder.beginStructure(TreeJsonDecoder.kt:258)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:29)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43)
2024-07-31 15:34:34.177 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:77)
2024-07-31 15:34:34.178 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:52)
2024-07-31 15:34:34.178 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.decodeSerializableValue(Tagged.kt:207)
2024-07-31 15:34:34.178 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder$decodeSerializableElement$1.invoke(Tagged.kt:279)
2024-07-31 15:34:34.178 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.tagBlock(Tagged.kt:294)
2024-07-31 15:34:34.178 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.decodeSerializableElement(Tagged.kt:279)
2024-07-31 15:34:34.178 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.internal.animation.AnimatedVectorN$Default$$serializer.deserialize(AnimatedVectorN.kt:36)
2024-07-31 15:34:34.178 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.internal.animation.AnimatedVectorN$Default$$serializer.deserialize(AnimatedVectorN.kt:36)
2024-07-31 15:34:34.178 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:77)
2024-07-31 15:34:34.178 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:52)
2024-07-31 15:34:34.178 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.TreeJsonDecoderKt.readJson(TreeJsonDecoder.kt:25)
2024-07-31 15:34:34.179 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.Json.decodeFromJsonElement(Json.kt:127)
2024-07-31 15:34:34.179 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.JsonContentPolymorphicSerializer.deserialize(JsonContentPolymorphicSerializer.kt:94)
2024-07-31 15:34:34.179 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:77)
2024-07-31 15:34:34.179 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:52)
2024-07-31 15:34:34.180 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.decodeSerializableValue(Tagged.kt:207)
2024-07-31 15:34:34.180 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder$decodeNullableSerializableElement$1.invoke(Tagged.kt:288)
2024-07-31 15:34:34.180 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.tagBlock(Tagged.kt:294)
2024-07-31 15:34:34.180 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.decodeNullableSerializableElement(Tagged.kt:286)
2024-07-31 15:34:34.180 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.internal.effects.EffectValue$Unsupported$$serializer.deserialize(EffectValue.kt:79)
2024-07-31 15:34:34.180 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.internal.effects.EffectValue$Unsupported$$serializer.deserialize(EffectValue.kt:79)
2024-07-31 15:34:34.180 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:77)
2024-07-31 15:34:34.180 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:52)
2024-07-31 15:34:34.180 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.TreeJsonDecoderKt.readPolymorphicJson(TreeJsonDecoder.kt:33)
2024-07-31 15:34:34.180 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:90)
2024-07-31 15:34:34.180 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:52)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.decodeSerializableValue(Tagged.kt:207)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder$decodeSerializableElement$1.invoke(Tagged.kt:279)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.tagBlock(Tagged.kt:294)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.decodeSerializableElement(Tagged.kt:279)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.encoding.CompositeDecoder$DefaultImpls.decodeSerializableElement$default(Decoding.kt:538)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.CollectionLikeSerializer.readElement(CollectionSerializers.kt:80)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.AbstractCollectionSerializer.readElement$default(CollectionSerializers.kt:51)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:36)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:77)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:52)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.decodeSerializableValue(Tagged.kt:207)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder$decodeSerializableElement$1.invoke(Tagged.kt:279)
2024-07-31 15:34:34.181 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.tagBlock(Tagged.kt:294)
2024-07-31 15:34:34.182 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.decodeSerializableElement(Tagged.kt:279)
2024-07-31 15:34:34.182 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.internal.effects.LayerEffect$UnsupportedEffect$$serializer.deserialize(LayerEffect.kt:32)
2024-07-31 15:34:34.182 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.internal.effects.LayerEffect$UnsupportedEffect$$serializer.deserialize(LayerEffect.kt:32)
2024-07-31 15:34:34.182 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:77)
2024-07-31 15:34:34.182 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:52)
2024-07-31 15:34:34.182 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.TreeJsonDecoderKt.readPolymorphicJson(TreeJsonDecoder.kt:33)
2024-07-31 15:34:34.182 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:90)
2024-07-31 15:34:34.182 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:52)
2024-07-31 15:34:34.182 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.decodeSerializableValue(Tagged.kt:207)
2024-07-31 15:34:34.182 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder$decodeSerializableElement$1.invoke(Tagged.kt:279)
2024-07-31 15:34:34.182 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.tagBlock(Tagged.kt:294)
2024-07-31 15:34:34.182 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.decodeSerializableElement(Tagged.kt:279)
2024-07-31 15:34:34.183 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.encoding.CompositeDecoder$DefaultImpls.decodeSerializableElement$default(Decoding.kt:538)
2024-07-31 15:34:34.183 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.CollectionLikeSerializer.readElement(CollectionSerializers.kt:80)
2024-07-31 15:34:34.183 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.AbstractCollectionSerializer.readElement$default(CollectionSerializers.kt:51)
2024-07-31 15:34:34.183 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:36)
2024-07-31 15:34:34.183 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43)
2024-07-31 15:34:34.183 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:77)
2024-07-31 15:34:34.183 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:52)
2024-07-31 15:34:34.183 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.decodeSerializableValue(Tagged.kt:207)
2024-07-31 15:34:34.183 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder$decodeSerializableElement$1.invoke(Tagged.kt:279)
2024-07-31 15:34:34.183 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.tagBlock(Tagged.kt:294)
2024-07-31 15:34:34.183 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.TaggedDecoder.decodeSerializableElement(Tagged.kt:279)
2024-07-31 15:34:34.183 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.internal.layers.ImageLayer$$serializer.deserialize(ImageLayer.kt:26)
2024-07-31 15:34:34.184 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.internal.layers.ImageLayer$$serializer.deserialize(ImageLayer.kt:26)
2024-07-31 15:34:34.184 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:77)
2024-07-31 15:34:34.184 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:52)
2024-07-31 15:34:34.184 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.TreeJsonDecoderKt.readPolymorphicJson(TreeJsonDecoder.kt:33)
2024-07-31 15:34:34.184 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:90)
2024-07-31 15:34:34.184 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:75)
2024-07-31 15:34:34.184 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
2024-07-31 15:34:34.184 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
2024-07-31 15:34:34.184 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
2024-07-31 15:34:34.184 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.encoding.CompositeDecoder$DefaultImpls.decodeSerializableElement$default(Decoding.kt:538)
2024-07-31 15:34:34.184 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.CollectionLikeSerializer.readElement(CollectionSerializers.kt:80)
2024-07-31 15:34:34.185 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.AbstractCollectionSerializer.readElement$default(CollectionSerializers.kt:51)
2024-07-31 15:34:34.185 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:36)
2024-07-31 15:34:34.185 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43)
2024-07-31 15:34:34.185 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
2024-07-31 15:34:34.185 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
2024-07-31 15:34:34.185 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
2024-07-31 15:34:34.185 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
2024-07-31 15:34:34.185 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.internal.Animation$$serializer.deserialize(Animation.kt:12)
2024-07-31 15:34:34.185 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.internal.Animation$$serializer.deserialize(Animation.kt:12)
2024-07-31 15:34:34.185 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
2024-07-31 15:34:34.185 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.serialization.json.Json.decodeFromString(Json.kt:107)
2024-07-31 15:34:34.186 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.LottieComposition$Companion.parse(LottieComposition.kt:349)
2024-07-31 15:34:34.186 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.JsonStringImpl$load$2$1.invokeSuspend(LottieCompositionSpec.kt:45)
2024-07-31 15:34:34.186 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.JsonStringImpl$load$2$1.invoke(Unknown Source:8)
2024-07-31 15:34:34.186 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.JsonStringImpl$load$2$1.invoke(Unknown Source:2)
2024-07-31 15:34:34.186 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.LruMap.getOrPutSuspend(LruMap.kt:53)
2024-07-31 15:34:34.186 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.LottieComposition$Companion.getOrCreate(LottieComposition.kt:332)
2024-07-31 15:34:34.186 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.JsonStringImpl$load$2.invokeSuspend(LottieCompositionSpec.kt:44)
2024-07-31 15:34:34.186 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.JsonStringImpl$load$2.invoke(Unknown Source:8)
2024-07-31 15:34:34.186 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.JsonStringImpl$load$2.invoke(Unknown Source:4)
2024-07-31 15:34:34.186 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:61)
2024-07-31 15:34:34.187 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:163)
2024-07-31 15:34:34.187 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source:1)
2024-07-31 15:34:34.187 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.JsonStringImpl.load-impl(LottieCompositionSpec.kt:43)
2024-07-31 15:34:34.187 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.JsonStringImpl.load(LottieCompositionSpec.kt:41)
2024-07-31 15:34:34.187 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at io.github.alexzhirkevich.compottie.LottieCompositionKt$rememberLottieComposition$1$composition$1.invokeSuspend(LottieComposition.kt:73)
2024-07-31 15:34:34.187 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
2024-07-31 15:34:34.187 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
2024-07-31 15:34:34.187 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:111)
2024-07-31 15:34:34.187 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:99)
2024-07-31 15:34:34.187 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:585)
2024-07-31 15:34:34.187 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:802)
2024-07-31 15:34:34.187 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:706)
2024-07-31 15:34:34.187 15729-15729 Verificati...okeSuspend com.demo.cm                          E  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:693)
```

**Comment by alexzhirkevich** [comment]
Thanks! The bug appears to be fixed already. The fix will be included in the next release

You can use this file instead if you need it asap: 
[fix.json](https://github.com/user-attachments/files/16438475/fix.json)

![image](https://github.com/user-attachments/assets/19b944f7-f293-4493-8aa4-994b125387ce)

**Comment by alexzhirkevich** [comment]
You can use this file instead if you need it asap: 
[fix.json](https://github.com/user-attachments/files/16438475/fix.json)

**Comment by Kale1314** [comment]
Thank you for your reply. I will try it out

**Comment by alexzhirkevich** [comment]
Should be fixed in `2.0.0-rc01`
